src/client.ts

```diff
@@ -366,9 +369,28 @@ export class ServiceClient {
       method: 'GET',
       headers: { Accept: 'application/json' },
     });
+
     await this.unwrap(response);
+    const json = await response.json();
 
-    return (await response.json()) as ProfilesListResponse;
+    return {
+      ...json,
+      data: json.data.map((profile: unknown) => {
+        if (isProfileMinimalReponse(profile)) {
+          return {
+            profile_id: profile.id,
+            profile_version: [0, 0, 0],
```

Comment:
Profile version is string.

```suggestion
            profile_version: '0.0.0',
```

Reply:
Thank you, this happens every time I decide not to write tests.